lua_examples/luaOTA/_doTick.lua

```diff
@@ -1,5 +1,4 @@
--- luacheck: globals self
-if (self.timer) then self.timer:stop() end--SAFETRIM
+--if (self.timer) then self.timer:stop() end--SAFETRIM
```

Comment:
If this isn't useful, please just remove it rather than comment it out.

Other than this little nit, this commit looks good to me.